sdk/core/azure-core/src/logging/logging.cpp

```diff
@@ -7,72 +7,41 @@
 #include <mutex>
 
 using namespace Azure::Core::Logging;
-using namespace Azure::Core::Logging::Details;
-
-class Azure::Core::Logging::Details::LogClassificationsPrivate {
-public:
-  static LogClassifications const LogClassificationsConstant(bool all)
-  {
-    return LogClassifications(all);
-  }
-
-  static bool IsClassificationEnabled(LogClassifications const& cls, LogClassification const& c)
-  {
-    return cls.m_all || (cls.m_classifications.find(c) != cls.m_classifications.end());
-  }
-};
+using namespace Azure::Core::Logging::Internal;
 
 namespace {
 std::mutex g_loggerMutex;
 LogListener g_logListener(nullptr);
+LogLevel g_logLevel = LogLevel::Verbose;
 
-LogClassifications g_logClassifications(
-    LogClassificationsPrivate::LogClassificationsConstant(true));
-
-LogListener GetLogListener(LogClassification const& classification)
+LogListener GetLogListener(LogLevel level)
```

Comment:
We don't expose this to customers, right?

Reply:
No, this is internal even to this only cpp file.